app/src/main/java/org/overlake/mat803/simon/GameFragment.java

```diff
@@ -79,18 +79,22 @@ private void changeColor(Button b) {
                 button = (ImageView) getActivity().findViewById(R.id.red);
                 button.setImageResource(R.drawable.ic_red_dark);
                 //button.setImageResource(R.drawable.ic_red);
+                break;
             case YELLOW:
                 button = (ImageView) getActivity().findViewById(R.id.yellow);
                 button.setImageResource(R.drawable.ic_yellow_dark);
```

Comment:
use binding - using findviewbyid for 4 switch cases is very resource inefficient